packages/drivers/routerlicious-driver/src/tokens.ts

```diff
@@ -87,8 +87,7 @@ export interface ITokenProvider {
 	 *
 	 * * Using the callback may have performance impact on the document creation process.
 	 *
-	 * * Any exceptions thrown in the callback would fail the creation workflow
```

Comment:
This comment was linking to an internal implementation detail, which also contains no further details on the topic.